operations/services.md

```diff
@@ -596,6 +502,230 @@ Updates price of a rate in the specified intervals. If the `CategoryId` is speci
 {}
 ```
 
+## Get all restrictions
+
+Returns all restrictions of the default service provided by the enterprise.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/restrictions/getAll`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D"
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+
+### Response
+
+```javascript
+{  
+   "Restrictions":[  
```

Comment:
Incorrect formatting of this block.
missing white spaces
-> `"Conditions": {`
-> ` "BaseRateId": null`